docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -0,0 +1,266 @@
+# Migration from v5 to v6
+
+<p class="description">This guide describes the changes needed to migrate the Date & Time Pickers from v5 to v6.</p>
+
+## Start using the alpha release
+
+In `package.json`, change the version of the data grid package to `next`.
+
+```diff
+-"@mui/x-date-pickers": "latest",
++"@mui/x-date-pickers": "next",
+```
+
+Using `next` ensures that it will always use the latest v6 alpha release, but you can also use a fixed version, like `6.0.0-alpha.0`.
+
+## Breaking changes
+
+Since v6 is a major release, it contains some changes that affect the public API.
+These changes were done for consistency, improve stability and make room for new features.
+Below are described the steps you need to make to migrate from v5 to v6.
+
+### Update the format of the `value` prop
+
+The `value` prop of the pickers now expects a parsed value.
+Until now, it was possible to provide any format that your date management library was able to parse.
+For instance, you could pass `value={new Date()}` when using `AdapterDayjs`.
+
+This behavior brought a lot of confusion and was removed.
+The format expected by the `value` prop is now the same as for any other prop holding a date.
+Here is the syntax to initialize a date picker at the current date for each adapter:
```

Comment:
I tied to rewrite it to clearly distinguish what is before and after.

I took inspiration from a random [migration guideline from Google](https://developer.chrome.com/docs/workbox/migration/migrate-from-v4/#default-precache-manifest-replacement-point) where I like the "Previously ....., In v5 ...." structure

```suggestion
Previously, it was possible to provide any format that your date management library was able to parse.
For instance, you could pass `value={new Date()}` when using `AdapterDayjs`. 
This behavior brought a lot of confusion.

In v6, the format expected by the `value` prop is the same as for any other prop holding a date.
Here is the syntax to initialize a date picker at the current date for each adapter:
```